readme.md

```diff
@@ -33,7 +33,107 @@ npm install colette --save
 
 ---
 ## Download
-- See the [`dist/`](https://github.com/20minutes/colette/dist/) 
+- See the [`dist/`](https://github.com/20minutes/colette/dist/)
 directory for ready-to-use css files.
-- If you want to use Colette as a kickstarter for custom stylus/css 
+- If you want to use Colette as a kickstarter for custom stylus/css
 projects, check [`src/`](https://github.com/20minutes/colette/assets/).
+
+---
+## Use gulp
+
+To compile your stylus, use `gulp` and `gulp-stylus`
```

Comment:
I would also add a few mentions to put things into their context:
1. mention that this section:
  - is meant for people using colette source files, and use gulp as a tasks manager,
  - gives an example of a gulp task to compile a project css file,
2. explain that colette directory must be added to the user/dev project, and that `assets` and `node_modules` folders are compulsory,
3. then give the structure of colette file:
````
  colette/
  |--assets/
  |  |-- [list folders]
  |--node_modules/
  |  |-- [list folders]
````